script/script.js

```diff
@@ -13,3 +13,232 @@ function eachClick() {
 innerButtons.forEach((item) => {
   item.addEventListener('click', eachClick);
 });
+
+const projectInfo = {
+  title: 'Keeping track of hundreds of components',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  technologies: ['Ruby on rails', 'css', 'JavaScript'],
+  image: '../images/project-image.svg',
+  live: 'https://yothu.github.io/my-portfolio/',
+  source: 'https://github.com/Yothu/my-portfolio',
+};
+
+const projectInfoArray = [
+  projectInfo.title,
+  projectInfo.description,
+  projectInfo.image,
```

Comment:
**Images aren't displaying well kindly work on it.**
![Screen Shot 2021-12-14 at 18 41 11](https://user-images.githubusercontent.com/22258274/146101880-3cb56bb0-e35b-4ed7-a53c-0c31708101eb.png)

Reply:
Just checked and the image appears, I also checked with in another computer just in case it only worked in mine's but it also worked.

Reply:
If your images are displaying well it's ok 😄